plugin/http/httptrace/httptrace.go

```diff
@@ -189,3 +189,12 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	r = r.WithContext(ctx)
 	h.handler.ServeHTTP(w, r)
 }
+
+func spanNameFromURL(prefix string, u *url.URL) string {
+	host := u.Hostname()
+	port := ":" + u.Port()
+	if port == ":" || port == ":80" || port == ":443" {
+		port = ""
+	}
+	return prefix + "." + host + port + u.Path
```

Comment:
How come we are no longer including the query string in here?

Reply:
query string can be very high cardinality, like `?userId={uuid}`. But then the `Path` could contain UUIDs as well, is it a good idea to include it in the span name?

Reply:
I agree with the removal of the query string. I view the Span name as something that should be "static", akin to Tag/Measure/View names.

Reply:
If we need the query string, it could be added as a annotation to the Span.